### PR TITLE
updated to show relative time only as far as yesterday

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

**Comment by wheresrhys** [comment]
I think we can get away with a minor release for this as the module description does say it is for formatting dates according to FT style.

File: main.js [hunk 1]
```diff
@@ -92,12 +107,13 @@ function ftTime(el) {
...
-	var interval = Math.round(((new Date()) - date) / 1000);
-	printer.innerHTML = interval < (365 * 60 * 60 * 24) ? timeAgo(date, interval) : format(date, 'date');
+	printer.innerHTML = ftTime(date);
+
 	el.title = format(date, 'datetime');
 	el.setAttribute('data-o-date-js', '');
 }
 
+
 function timeAgo (date, interval) {
```
Comment (AlbertoElias) [inline main.js]: can't we just reduce the logic inside this function and move your logic in ftTime here or vice versa?

File: main.js [hunk 2]
```diff
@@ -79,7 +79,22 @@ function autoUpdate () {
 }
 
 
-function ftTime(el) {
+function ftTime(dateObj) {
+	var now = new Date();
+	var interval = Math.round((now - dateObj) / 1000);
+	var dateString;
+
+	if (interval < 24 * 60 * 60 && now.getDay() === dateObj.getDay()) {
```
Comment (AlbertoElias) [inline main.js]: Probably useful to add a comment before each if briefly mentioning what you're checking as it's something that you need to read carefully to understand and it would be much faster with a short comment

File: main.js (hunk 1 above)
Comment (AlbertoElias) [inline main.js]: It would probably be a major release, but I don't think that's an issue. Or, as you've already released this, you could make another pr cleaning that up and we can make a major with that

File: main.js (hunk 2 above)
Comment (wheresrhys) [inline main.js]: yep

File: main.js (hunk 1 above)
Comment (wheresrhys) [inline main.js]: I quite like the separation of concerns between construction of the date string and working out what date to use for the element

Comment (AlbertoElias) [inline main.js]: But your basically not using timeAgo anymore, only the minute and hours bit. You moved the _a day ago_ and the rest to the other function, so if you want it to be separated, you can move your new code into the timeAgo function for example. But you've got the timeAgo function separated into two currently
